File: courses/CS50-2023/week-1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //requesting input
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //printing rows
    for (int n = 0; n < height; n++)
    {
        //left side
        //printing spaces
        for (int s = height - 1; s > n; s--)
        {
            printf(" ");
        }
        //printing bricks
        for (int i = 0; i <= n; i++)
        {
            printf("#");
        }

        //printing distance in the middle
        printf("  ");

        //right side
        for (int i = 0; i <= n; i++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: courses/CS50-2023/week-6/sentimental-mario-more/mario.py
# TODO

from cs50 import get_int

# prompting user for height
while True:
    height = get_int("Height: ")

    # reprompting if input invalid
    if height > 0 and height <= 8:
        break


# printing piramide
for i in range(1, height + 1):
    hashes = "#" * i
    spaces = " " * (height - i)

    print(spaces + hashes + "  " + hashes)
